web/modules/custom/joinup_core/src/Form/RdfEntityUriWebConverterForm.php

```diff
@@ -37,14 +38,19 @@ public function validateForm(array &$form, FormStateInterface $form_state) {
     global $base_url;
     parent::validateForm($form, $form_state);
 
-    $id = $path = $form_state->getValue('rdf_entity_id');
+    $id = $path = trim($form_state->getValue('entity_id'), " /");
 
     // An absolute URL.
     if (UrlHelper::isExternal($id)) {
       // Potential RDF entity ID.
       if (!UrlHelper::externalIsLocal($id, $base_url)) {
         if (Rdf::load($id)) {
           $form_state->set('passed_as', 'id');
+          $form_state->set('entity_type', 'rdf_entity');
+        }
+        elseif (RdfTerm::load($id)) {
```

Comment:
This is now hardcoded. Normally, we would iterate over all content entities that have SPARQL storage and try to load with each, one by one. That would generalize to any new SPARQL entity type that might be added in the future. But for now I agree that this is a very "Joinup centric" functionality and we can go with the hardcoding.